modules/debug_cmd/debug_cmd.c

```diff
@@ -143,30 +143,34 @@ static int cmd_play_file(struct re_printf *pf, void *arg)
 {
 	struct cmd_arg *carg = arg;
 	struct config *cfg;
-	const char *filename = carg->prm;
+	char filename[4096] = { 0 };
```

Comment:
This could be rewritten using `char filename[strlen(carg->prm)]` if `carg->prm` is set, but that triggers a warning about initialising a variable length array.

Reply:
Pls avoid such large stack buffers!